frontend/src/stores/user.js

```diff
@@ -70,13 +70,16 @@ class userStore {
         this.pinCode[position] = value;
         return true;
     };
-    setCredentials = (userData) => {
+    setCredentials(userData) {
```

Comment:
This a little misleading now that there's some of the items as methods and some as fields. Could this be unified?

Reply:
yes, pushed